src/events/EventsSection.tsx

```diff
@@ -100,6 +179,35 @@ export const EventsSection = () => {
     </>
   );
 
+  const clearSelection = (e: any) => {
+    e.stopPropagation();
+    setSelectedEvents([]);
+    setValue("eventTypes", []);
+  };
+
+  const deleteIndividualSelection = (e: any, chip: string) => {
+    const updatedEventSelection = selectedEvents.filter((obj) => obj !== chip);
+    setSelectedEvents(updatedEventSelection);
+    setValue("eventTypes", updatedEventSelection);
+    e.stopPropagation();
+  };
+
+  const chipGroupComponent = () => {
+    return (
+      <ChipGroup>
+        {(selectedEvents || []).map((chip, index) => (
```

Comment:
`selectedEvents` always has a value, so it can be mapped directly.
```suggestion
        {selectedEvents.map((chip, index) => (
```

Reply:
true - thanks!